react/features/base/premeeting/components/web/ConnectionStatus.tsx

```diff
@@ -145,10 +158,15 @@ const CONNECTION_TYPE_MAP: {
  * @param {IProps} props - The props of the component.
  * @returns {ReactElement}
  */
-function ConnectionStatus({ connectionDetails, t, connectionType }: IProps) {
+const ConnectionStatus = React.memo(({ connectionDetails, t, connectionType, _runPreCallTest }: IProps) => {
```

Comment:
The prejoin screen does a bunch of re-renders, this was just meant to prevent some on this component. 
The component was refactored it to useSelector and useDispatch as requested on another comment, so this got removed.